avvoenv.1.ronn

```diff
@@ -92,6 +96,14 @@ current service, or canonicalise the name given with the `--service` option.
     range eg `[0-9]` or `[a-z]`
     `[!...]` is the inverse of `[...]`
 
+  * `-p`, `--app-id` [<app-id>]:
+    Authenticate with Vault via app-id. If the <app-id> argument is provided
+    it will override the `VAULT_APP_ID` environment variable.
+
+  * `-r`, `--app-user` [<app-user>]:
```

Comment:
The `[]` are used to indicate that the argument isn't required (eg the option can be used as a flag), which isn't the case here.